lib/responses.js

```diff
@@ -312,14 +330,31 @@ function unauthorized(res) {
  * @param {object} res
  */
 function clientError(err, res) {
-  log('error', err.message, { stack: err.stack, url: _.get(res, 'locals.url') });
   // They know it's a 400 already, we don't need to repeat the fact that its an error.
   const message = removePrefix(err.message, ':'),
     code = 400;
 
+  log('warn', err.message, { code, stack: err.stack, url: _.get(res, 'locals.url') });
```

Comment:
could use `logLevelFromStatusCode` here incase in the future you want to change behavior for 400s